legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py

```diff
@@ -891,11 +891,11 @@ def test_unpaid_filing(session, client, jwt):
                     }
       },
      HTTPStatus.OK, '2017-10-01'
-    ),
+     ),
```

Comment:
This is really nitpicky, but I am curious if this extra space (repeated two times, below as well (918 and 942)) is needed or is a typo.

Reply:
It's auto formatted by vs code. If you see rest of the parameter object in this function you will see the same spacing